awx/main/tasks.py

```diff
@@ -1201,6 +1250,9 @@ def run(self, pk, **kwargs):
             self.build_extra_vars_file(self.instance, private_data_dir)
             args = self.build_args(self.instance, private_data_dir, passwords)
             cwd = self.build_cwd(self.instance, private_data_dir)
+            # Must prepare profiling params before isolation, so that data dir can be whitelisted
```

Comment:
probably don't need this whitelist comment anymore

Reply:
removed